apps/ejabberd/src/ejabberd_sm.erl

```diff
@@ -964,14 +968,14 @@ process_iq(From, To, Acc, Packet) ->
                     ResIQ = Module:Function(From, To, IQ),
```

Comment:
According to **Elvis**:
> Remove the dynamic function call on line 968. Only modules that define callbacks should make dynamic calls.

Reply:
Wontfix - would require a rewrite of the whole iq handling machinery, which is not within current scope of work.